setup-slave.sh

```diff
@@ -45,6 +46,18 @@ if [[ $instance_type == r3* ]]; then
   fi
 fi
 
+if [[ $instance_type == i2* ]]; then
```

Comment:
Can we move this to the same block as `r3` i.e. have `$instance_type == r3* || $instance_type == i2*`